pom.xml

```diff
@@ -156,7 +156,7 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-enforcer-plugin</artifactId>
-          <version>3.0.0-M3</version>
+          <version>3.1.0</version>
```

Comment:
When you bump the parent POM, it might no longer be necessary to specify a version or have a dependencyManagement entry at all, for plugins whose version and config are identical (or older than) what's specified in the parent POM.

Reply:
I removed the version in ee1e9f7

Reply:
I'll try removing the entry entirely

Reply:
Removed in 0cbfa26, enforcer runs with the correct version so should be good to go.